CHANGELOG.asciidoc

```diff
@@ -25,6 +25,7 @@ NEED AND IMAGE
 
 This release also includes changes from <<release-3-3-2, 3.3.2>>.
 
+* `min()` and `max()` now support all types implementing `Comparable`.
```

Comment:
this should be on 3.4.0 right?

Reply:
It is, and I have no idea why you get that wrong in all my PRs :)